wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs

```diff
@@ -22,6 +23,14 @@ use serde_json::to_string;
 use types::types::ExecutionTipState;
 use wevm_borsh::block::BorshSealedBlockWithSenders;
 
+pub fn get_network_tag() -> &'static str {
+    let devnet_flag = env::var("DEVNET").unwrap_or(String::from("false")).to_lowercase();
+    if devnet_flag == "true" {
+        return "Devnet"
```

Comment:
"Devnet v0.1.0" to align with Alphanet naming

Reply:
hmmmm
or it is 0.2.0??

Reply:
cause it is a next version

Reply:
0.2.0 sgtm!